droidlet/dashboard/web/src/components/Timeline/README.md

```diff
@@ -0,0 +1,29 @@
+# Timeline for Droidlet
+
+This component consists of an agent activity visualizer for the Droidlet dashboard app. Enable this when running the
+agent using the flags `--enable_timeline --log_timeline`.
```

Comment:
You should document the two flags a little bit more, like just use '--enable_timeline' is enough to see the timeline in dashboard, and '--log_timeline' will save it to a file which can further be used to recreate the timeline.

Reply:
Thanks for the feedback! I've added this in [67e8bf5](https://github.com/facebookresearch/droidlet/pull/593/commits/67e8bf5a937110949624406281e8cba8ec240672), lmk if that looks okay!